### PR TITLE
feat: #WB-1985, create wrapper resize media

### PR DESCRIPTION
**Pull request by Headan**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

>

Ajout d'un wrapper pour resize un media

## Type of change

Ticket: #WB-1985
Lien : https://edifice-community.atlassian.net/browse/WB-1985

Please check options that are relevant.

- [x] Chore (PATCH)
- [ ] Doc (PATCH)
- [ ] Bug fix (PATCH)
- [ ] New feature (MINOR)
- [ ] Breaking change (MAJOR)

> PATCH: refactor, internal or non-breaking change which fixes an issue
>
> MINOR: non-breaking change which adds functionality
>
> MAJOR: fix or feature that would cause existing functionality to not work as expected

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: frontend/src/styles/table.scss [hunk 1]
```diff
@@ -63,3 +63,36 @@
   cursor: ew-resize;
   cursor: col-resize;
 }
+
```
[the remark this answers is not in the document]
Comment (Headan) [reply frontend/src/styles/table.scss]: @ccreusat Déplacé dans edifice-bootstrap => https://github.com/opendigitaleducation/edifice-bootstrap/pull/36

File: frontend/src/components/Tiptap/ImageEditMenu.tsx
[the remark this answers is not in the document]
Comment (Headan) [reply frontend/src/components/Tiptap/ImageEditMenu.tsx:1]: @ccreusat J'ai eu un petit soucis de type du coup j'appelle BubbleMenu dans tiptap playground et je lui donne en children le nouveau composant créé dans edifice-ui

File: frontend/src/components/Tiptap/VideoNodeView.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import { ReactNodeViewRenderer } from "@tiptap/react";
+
+const VideoResize = (Component: any) =>
+  Video.extend({
+    addNodeView() {
+      return ReactNodeViewRenderer(Component);
+    },
+  });
+
+export default VideoResize;
```
Comment (ccreusat) [inline frontend/src/components/Tiptap/VideoNodeView.tsx:11]: @Headan doublon d'export > named export suffisant

File: frontend/src/components/Tiptap/AttachmentNodeView.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import { ReactNodeViewRenderer } from "@tiptap/react";
+
+const AttachmentView = (Component: any) =>
+  AttachmentExt.extend({
+    addNodeView() {
+      return ReactNodeViewRenderer(Component);
+    },
+  });
+
+export default AttachmentView;
```
Comment (ccreusat) [inline frontend/src/components/Tiptap/AttachmentNodeView.tsx:11]: @Headan doublon d'export > named export suffisant

File: frontend/src/components/Tiptap/ImageNodeView.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import { ReactNodeViewRenderer } from "@tiptap/react";
+
+const ImageResize = (Component: any) =>
+  ImageExtend.extend({
+    addNodeView() {
+      return ReactNodeViewRenderer(Component);
+    },
+  });
+
+export default ImageResize;
```
Comment (ccreusat) [inline frontend/src/components/Tiptap/ImageNodeView.tsx:11]: @Headan doublon d'export > named export suffisant

**Review by ccreusat — CHANGES_REQUESTED** [review]
@Headan les composant `xNodeView` sont à externaliser dans `edifice-ui/react/tiptap`. Il n'ont pas vocation à rester dans tiptap-playground.

**Review by ccreusat — APPROVED** [review]

**Pull request merged**